### ISSUE TITLE
Localization 

### ISSUE THREAD

**Issue by KovalevArtem** [mangerlahn/Latest#110]
It is sad that the application is only in En, De, Fr- I propose to translate.
I am ready to help in the translation into Russian.
(For example, you can use Weblate, or any other service. It is completely free for open source projects.)

![image](https://user-images.githubusercontent.com/36500228/98449379-2b20e180-2144-11eb-8870-57606b640a77.png)

**Comment by mangerlahn** [comment]
Thank you very much for bringing this up! I will setup Weblate and reach out, when I have something ready!

**Comment by mangerlahn** [comment]
I think I will go with Transifex instead, but I need to prepare the project a bit. Trying to do that in the coming days.

**Comment by KovalevArtem** [comment]
**Fine** - Weblate
**Okay** - Crowdin
**Poorly** - Transifex, POEditor, Lokalise, OneSky

_This is my subjective assessment of the convenience of these services. But please don't choose Transifex..._

**Comment by mangerlahn** [comment]
Haha, okay, then sorry about that!

I have looked briefly into Weblate and as far as I can see, it is based on Git. This would take me some restructuring of the project, but that is totally fine. 
One thing I am concerned about is that Weblate does not seem to support `.stringsdict` files, which are used to localize plurals in Xcode. An alternative would be to use `.xliff` files. Xcode can export/import them and I use them to translate Latest into German. However, we kind-of loose the automatisms Weblate provides, as I would need to remember to put up-to-date xliffs into the repo every time I change something in the UI. With Transifex I would just dump XLIFF files there and re-import them from time to time. I am not attached to them at all and would happily use Weblate. 

Have you worked on other iOS / macOS apps with Weblate and could share their process? I am grateful for any help on Latest, so I want to make sure the process is as easy as possible for everyone involved :)

**Comment by KovalevArtem** [comment]
@comradekingu We need your help 😊

**Comment by comradekingu** [comment]
(I am only a WL contributor) https://docs.weblate.org/en/latest/vcs.htm There are some WL macOS projects here and there. I have contributed to the ones I found even if I don't use macOS, though I don't work on CoC projects. https://docs.weblate.org/en/latest/formats.html?highlight=xliff is supported. The main difference between services is the amount of data they leak, steal or sell https://www.transifex.com/legal/cookies-policy/ and whether they are as-a-service only. Fortunately usability lies with the prominent libre software alternative IMO. :)
